From e384d1ca13a5cc76e238ee10c3af7b493e6ac28f Mon Sep 17 00:00:00 2001
From: n3hrox <n3hrox@gmail.com>
Date: Sun, 3 Nov 2019 21:05:24 +0100
Subject: [PATCH] add build patch

---
 dvc/_build.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dvc/_build.py b/dvc/_build.py
index 7bf1d88..84c77c8 100644
--- a/dvc/_build.py
+++ b/dvc/_build.py
@@ -1 +1 @@
-PKG = "pip"
+PKG = "conda"
-- 
2.17.1
